packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1900,12 +2102,19 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         // Prevent this call from having any effect unless in a custom-set VM frame
         require(msg.sender == address(0));
```

Comment:
What's the message sender by default when you send an L1 => L2 message? Not the zero address, right?

Reply:
That's a really interesting question and truthfully I have no idea (cc @tynes ) but even if it were, I think this shouldn't be breakable, since L1=>L2 messages are still forced to enter via `run`.